rocky/views/ooi_list.py

```diff
@@ -77,16 +76,18 @@ def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
             return self._delete_oois(selected_oois, request, *args, **kwargs)
 
         if action == PageActions.UPDATE_SCAN_PROFILE.value:
-
-            return self._set_scan_profiles(selected_oois, request, *args, **kwargs)
+            scan_profile = request.POST.get("scan-profile")
+            level = CUSTOM_SCAN_LEVEL[scan_profile]
+            if level.value == "Inherit":
```

Comment:
It would be better if these values are lowercased, just to keep the API (eventhough its a web-interface driven POST) sane.